.travis.yml

```diff
@@ -9,7 +9,7 @@ matrix:
       language: cpp
       env: VS15=false RS_CPP_TEST=true
       script:
-        - cmake .. -G "Visual Studio 15 2017 Win64" -DBUILD_UNIT_TESTS=true -DBUILD_EXAMPLES=false -DBUILD_GRAPHICAL_EXAMPLES=false -DBUILD_WITH_TM2=false -DFORCE_RSUSB_BACKEND=true
+        - cmake .. -G "Visual Studio 15 2017 Win64" -DBUILD_UNIT_TESTS=true -DBUILD_EXAMPLES=false -DBUILD_GRAPHICAL_EXAMPLES=false -DBUILD_WITH_TM2=false -DFORCE_RSUSB_BACKEND=true -DENABLE_RS_AUTO_UPDATER=true
```

Comment:
Shouldn't the default be to enable the auto-updater?

Reply:
This is done by request.
Sergey?

Reply:
Sergey answer: "binary distributions will have it, build from source - you have to enable it"